lib/config/config-file.js

```diff
@@ -176,6 +176,40 @@ function loadPackage(filePath) {
 }
 
 /**
+ * Loads a JavaScript configuration from a plugin.
+ * @param {string} filePath The plugin name to load.
+ * @returns {Object} The configration object from the plugin.
+ * @throws {Error} If the package cannot be read.
+ * @private
+ */
+function loadPlugin(filePath) {
+    var parts = filePath.split("/"),
+        pluginName = parts[0],
+        configName = parts[1];
+    debug("Loading config plugin: " + filePath);
+    try {
+        if (!configName) {
+            debug("Error reading package: " + filePath);
+            throw new Error("No configuration name specified");
+        }
+        var plugin = require(pluginName);
```

Comment:
Hmm.. issue here is timing. In order to use `Plugins.get()` you need to call `Plugins.loadAll(config.plugins)` first. But we don't have config yet at this point.